doc/nvim-tree-lua.txt

```diff
@@ -440,6 +440,7 @@ applying configuration.
       },
       notify = {
         threshold = vim.log.levels.INFO,
+        handler = "default",
```

Comment:
Do we need that default? Can't it be done "behind scenes" and only used if set to function?

Reply:
I tried to do it in a similar way to how other options were done (e.g. window picker). I am happy to change it if you want.

Reply:
I think it would make sense to have default option implied, with ability to override it.

Reply:
Done, let me know if this is what you had in mind.

Reply:
That's kind of a break from existing - everything is specified in `DEFAULT_OPTS`. It's useful if only for the "default schema" that users can base their configuration on.

That's how my local dev harness works: put all the default options in the init lua setup call.

@gegoune are you OK to back that change out? Happy to revisit later; trimming the default options to a skeleton with no values would be less overwhelming.

Reply:
Missed that point. Of course. @ghostbuster91 Please reinstate state before my comment and accept my apologies for causing you extra work.

Reply:
No worries, luckily it was just a small change :)